samples/responses/v1/DiscoverAppliancesResponse.json

```diff
@@ -0,0 +1,397 @@
+{
+  "header": {
+    "messageId": "01ebf625-0b89-4c4d-b3aa-32340e894681",
+    "name": "DiscoverAppliancesResponse",
+    "namespace": "Alexa.ConnectedHome.Discovery",
+    "payloadVersion": "2"
+  },
+  "payload": {
+    "discoveredAppliances": [
+      {
+        "actions": [
+          "turnOn",
+          "turnOff"
+        ],
+        "applianceId": "ECHO_Switch_All_Lights",
+        "manufacturerName": "OpenHAB",
+        "modelName": "Switch",
+        "version": "1",
+        "friendlyName": "All Lights",
+        "friendlyDescription": "Switch All Lights via OpenHAB",
```

Comment:
Sorry all the samples/responses/ were not added by me, and now that I think about are not very useful here, I will remove completely.